sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ClientEncryptionPolicy.java

```diff
@@ -40,9 +40,25 @@ public final class ClientEncryptionPolicy {
      */
     @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public ClientEncryptionPolicy(List<ClientEncryptionIncludedPath> paths) {
-        this.validateIncludedPaths(paths);
-        this.includedPaths = paths;
         this.policyFormatVersion = 1;
+        this.validateIncludedPaths(paths, policyFormatVersion);
+        this.includedPaths = paths;
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param paths list of path of the item that need encryption along with path-specific settings.
+     * @param policyFormatVersion version of the client encryption policy definition. Current supported versions are 1 and 2. Default version is 1.
+     */
+    @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ClientEncryptionPolicy(List<ClientEncryptionIncludedPath> paths, int policyFormatVersion) {
+        if (policyFormatVersion > 2 || policyFormatVersion < 1) {
```

Comment:
I think we can make policyFormatVersion immutable (so that it is initialized only once per ClientEncryptionPolicy).